application/app/lib/project_name_generator.rb

```diff
@@ -0,0 +1,40 @@
+# frozen_string_literal: true
+
+# ProjectNameGenerator generates randomized, domain-relevant project names
+# by combining a scientific or HPC-related adjective and noun.
+# Optionally, a numeric token and custom delimiter can be included.
+# Useful for assigning default names to projects in research, HPC, or data-driven environments.
+class ProjectNameGenerator
```

Comment:
This class is simply to add a random name on project creation like Docker does, but with HPC related terms.

This is to avoid having the typical create form for projects. A project is created by clicking on create project, then the data can be updated

Reply:
great idea